evap/staff/forms.py

```diff
@@ -385,6 +387,7 @@ class Meta:
 
     def __init__(self, *args, **kwargs):
         semester = kwargs.pop("semester", None)
+        self.operation = kwargs.pop("operation", "save")
```

Comment:
I think this is clearer as a `operation: Literal["approve", "save"] = "save"` argument, outside of `kwargs` - in fact, should there be a default? Because this is effectively a toggle for one of the checks, it's a bit odd to have it off-by-default

Reply:
That doesn't work due to the forms structure